### ISSUE TITLE
This nodes are not variables?

### ISSUE THREAD

**Issue by knowledgenude** [armory3d/armory#1885]
![Screenshot from 2020-09-23 17-20-43](https://user-images.githubusercontent.com/63247726/94065183-29cc6d00-fdc1-11ea-8ed7-7793c55f3b05.png)

**Comment by MoritzBrueckner** [comment]
Nope, they aren't but I agree that they probably should be used as variables.

Regarding the inputs: why shouldn't they have inputs? Actually, this is much easier to use than the `Set Variable` reverse thingy node.

**Comment by knowledgenude** [comment]
I mean if we put a text box like we have in On Event, and the same style for variables like object (i am not sure if there is a way to do that for objects). 

For example, in the Collection node it have a way to put a "default" value but not a input properly. 

I think this would leave the things more clean, but i am not sure about the usability of these inputs.

**Comment by MoritzBrueckner** [comment]
> I mean if we put a text box like we have in On Event

You can have both an input socket and a textbox/property search field :)

Variable nodes act as a "storage" where you can save and read values from. The property search field is only the initial, default value. I think we should actually add inputs to variable nodes as well and extend the entire variable functionality. Of course this still needs to be documented(!) and it actually might be a bit confusion to have both inputs and the `Set Variable` node.
